Autosubmit is not compatible with rocrate>0.14.0

Hi @dbeltrankyl , can you try to reinstall rocratepy locally? I think they released a new version today, and it could be causing this test to fail.

_Originally posted by @kinow in https://github.com/BSC-ES/autosubmit/issues/2939#issuecomment-4236310743_

Confirmed issue cc: @kinow

Looking at https://github.com/BSC-ES/autosubmit/pull/2939#issuecomment-4235635250 and the error message:
```
FAILED test/integration/provenance/test_rocrate.py::test_rocrate_validator - AssertionError: [CheckIssue(severity=Severity.REQUIRED, check=<rocrate_validator.requirements.shacl.checks.SHACLCheck object at 0x7fb1273dad50>, message=The RO-Crate metadata file descriptor MUST have a `conformsTo` property with the RO-Crate specification version)]
assert not True
 +  where True = has_issues()
 +    where has_issues = ValidationResult(passed=False,issues=[CheckIssue(severity=Severity.REQUIRED, check=<rocrate_validator.requirements.shacl.checks.SHACLCheck object at 0x7fb1273dad50>, message=The RO-Crate metadata file descriptor MUST have a `conformsTo` property with the RO-Crate specification version)]).has_issues
= 1 failed, 419 passed, 18 skipped, 3 xfailed, 59917 warnings in 472.60s (0:07:52) =
Error: Process completed with exit code 1.
```

This is probably happening because ro-crate-py 0.15.0 produces crates conforming to 1.2 spec by default, but rocrate-validator doesn't yet have support for RO-Crate 1.2, only 1.1. Therefore, when you put a 1.2 crate into rocrate-validator, it falls back onto the 1.1 checks, and then raises an error because the crate doesn't conform to the version it's expecting.

If you don't need the 1.2 features, you could work around this by using `version=1.1` when initialising a crate with ro-crate-py:
```
crate = ROCrate(version="1.1")
```

Alternatively, if you do want to use 1.2 crates, you could instead disable the relevant check in the validation. You do this by including the check's ID (which is `ro-crate-1.1_5.3`, findable with the CLI command `rocrate-validator profiles describe ro-crate-1.1 -v`) in the `ValidationSettings` object passed to rocrate-validator:
```
settings = services.ValidationSettings(
            rocrate_uri="path/to/crate",
            skip_checks = ["ro-crate-1.1_5.3"]
        )
result = services.validate(settings)
```

--- a/autosubmit/provenance/rocrate.py
+++ b/autosubmit/provenance/rocrate.py
@@ -362,7 +362,7 @@ def create_rocrate_archive(
 
     _init_mimetypes()
 
-    crate = ROCrate(gen_preview=True)
+    crate = ROCrate(version="1.1", gen_preview=True)
     crate.name = expid
     # Fetch the experiment description from the main database
     crate.description = get_experiment_description(expid)[0][0]

